packages/core/src/components/StepList/StepLink.jsx

```diff
@@ -0,0 +1,39 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export class StepLink extends React.PureComponent {
+  handleClick(event) {
+    if (this.props.onEnterStep) {
+      event.preventDefault();
+      this.props.onEnterStep(this.props.route, this.props.stepId);
+    }
+  }
+
+  render() {
+    const { route, screenReaderText, className, children } = this.props;
+    const onClick = event => this.handleClick(event);
+    return (
+      <a href={route} onClick={onClick} className={className}>
+        {children}
+        <span className="ds-u-visibility--screen-reader">
+          {' '}
+          {screenReaderText}
```

Comment:
I would expect this text to read out as "Edit a tax year" or "Edit household details". It might take some pruning and careful stemming of the existing text strings, but will be worth doing to give assistive device users a clear call to action with natural language.

Reply:
Because of the differences in grammar between languages, I'm not comfortable composing sentences with smaller translated parts, so this would require us to pass multiple strings for each step, which is why I went with this method. We'd need to pass "Resume selecting a tax year", "Edit tax year", "Start selecting a tax year", "Select a tax year", which seems like a lot

Reply:
Is there a simpler way we could do this without it being jarring for users?